docs/UserGuide.adoc

```diff
@@ -196,16 +196,25 @@ _{explain how the user can reschedule their orders when customers notify them th
 
 == Command Summary
 
-* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
-e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-* *Clear* : `clear`
-* *Delete* : `delete INDEX` +
-e.g. `delete 3`
-* *Edit* : `edit TRANSACTION_ID FIELD_TO_CHANGE CHANGES` +
-e.g. `edit 0123456789 n/James Lee p/9999 8888`
-* *Find* : `find KEYWORD [MORE_KEYWORDS]` +
-e.g. `find James Jake`
+* *Add* : `add` `tid/TRANSACTION_ID` `n/NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]` +
+e.g. `add` `tid/0123456789` `n/Eng Xuan En` `a/Tampines St 84 Blk 877 S520877 #01-123` `p/87654321` `ts/2020-02-20 1300` `w/Yishun industry` `pm/cash` `c/please knock the door three times :D` `type/heavy`
+* *Clear* : `clear` `[FLAG]` +
+e.g. `clear` `-f`
+* *Delete* : `delete` `INDEX` +
+e.g. `delete` `2`
+* *Done* : `done` `INDEX` +
+e.g. `done` `2`
+* *Edit* : `edit` `INDEX` `PREFIX/VALUE` +
+e.g. `edit` `2` `n/Xuan En`
+* *Exit* : `exit`
+* *Find* : `find` `flag` `KEYWORD` +
+e.g. `find` `-t` `ac1e345x7s`
 * *Help* : `help`
-* *Import* : `import FILE NAME` +
-e.g. `import orders.csv`
-* *List* : `list`
+* *Import* : `import` `FILE_NAME` +
+e.g. `import` `orders.csv`
+* *List* : `list` `[DONE_STATUS]` +
+e.g. `list` `done`
+* *Return* : `return` `tid/TRANSACTION_ID` `n/NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `c/COMMENTS_BY_CUSTOMER` `type/TYPE_OF_ITEM` +
+e.g. `return` `tid/ac17s2a` `n/BOBBY TAN` `a/123 Delta Road #03-333, Singapore 123456` `p/91230456` `ts/12-12-2020` `w/Jurong Warehouse` `pm/visa` `c/NIL` `type/glass`
+* *Sort* : `sort` `FLAG` `NAME/TIMESTAMP/TRANSACTION_ID` +
+e.g. `sort` `-d` `ts/`
```

Comment:
The sort command is based on any prefix that is related to the add command, not limited to name, timestamp and transaction id.
![image](https://user-images.githubusercontent.com/38811217/75689261-db983500-5cdb-11ea-9228-e5abe6104ad6.png)

Reply:
I think can do
`sort` `[ORDER FLAG]` `ORDER_ATTRIBUTE_PREFIX`, then link to the sort section for more details

Reply:
Resolved